package.json

```diff
@@ -32,8 +32,8 @@
   },
   "scripts": {
     "prepublish": "babel src --out-dir dist",
-    "lint": "eslint .",
-    "tape": "babel-tape-runner 'src/**/__tests__/*.js'",
+    "lint": "eslint src",
```

Comment:
It looks like we can also explicitly set the .gitignore with
`eslint --ignore-path .gitignore`
?

Shall I amend the PR?